stingray/crossspectrum.py

```diff
@@ -570,8 +705,8 @@ def _make_auxil_pds(self, lc1, lc2):
             Two light curves used for computing the cross spectrum.
         """
         if lc1 is not lc2 and isinstance(lc1, Lightcurve):
-            self.pds1 = Crossspectrum(lc1, lc1, norm='none')
-            self.pds2 = Crossspectrum(lc2, lc2, norm='none')
+            self.pds1 = Crossspectrum(lc1, lc1, norm=self.norm)
+            self.pds2 = Crossspectrum(lc2, lc2, norm=self.norm)
```

Comment:
It was confusing because the pdss were unnormalized. Now, it's consistent: the auxiliary pdss have `unnorm_power` and `power`, and `power` is normalized as the main crossspectrum